src/series11/SwapArray.java

```diff
@@ -0,0 +1,29 @@
+package series11;
+
+import java.util.Scanner;
+
+public class SwapArray {
+ public static void main(String[] args)
+ {
+	 Scanner sc = new Scanner(System.in);
+	 System.out.println("Enter the array size");
+	 int n =sc.nextInt();
+	 int[] array = new int[n];
+	 System.out.println("Enter the array elements");
+	 for(int i = 0; i < n; i++)
+	 {
+		 array[i] = sc.nextInt();
+	 }
+	 for(int i = 0; i < n; i++)
+	 {
+	 int temp = 0;
```

Comment:
Fix the indentation for this problem. That is: The code inside for loop should not be on the same line as the for loop itself.